tests/contrib/django/django_app/urls.py

```diff
@@ -32,6 +34,11 @@ def authenticated_view(request):
     return HttpResponse(status=200)
 
 
+def shutdown(request):
+    tracer.shutdown()
```

Comment:
why?

Reply:
at the end of a test case we query this endpoint in order to have the traces flushed out. This removes the need to `time.sleep()` to receive traces

Reply:
comment would be nice :)

I know there is one down below, but this probably a weird endpoint to come across in code unless you know of the daphne code/comment